magmi/inc/magmi_engine.php

```diff
@@ -409,7 +409,7 @@ public function getExceptionTrace($tk, &$traces)
                 }
             }
             // simple max trace depth fix
-            if (++$counter % 20) {
+            if (++$counter % 20 == 0) {
```

Comment:
lets have === just for a sake of being clean.

Reply:
Done :)